package.json

```diff
@@ -1,22 +1,32 @@
 {
-  "name": "discordbot",
+  "name": "projectbot",
   "version": "1.0.0",
-  "description": "",
-  "main": "main.js",
+  "description": "Bot for managing TTRPG projects",
+  "author": "@jermainelecky",
+  "license": "MIT",
+  "private": true,
+  "main": "src/index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node src/index.js",
+    "lint": "eslint src",
+    "lint:fix": "eslint src --fix"
   },
-  "prettier":{
-    "useTabs": true,
-    "semi": true,
-    "singleQuote": true
-  },
-  "author": "Jermaine",
-  "license": "ISC",
   "dependencies": {
     "@discordjs/builders": "^0.13.0",
     "@discordjs/rest": "^0.4.1",
     "discord-api-types": "^0.33.0",
-    "discord.js": "^13.7.0"
+    "discord.js": "^13.7.0",
+    "dotenv": "^16.0.1",
+    "lodash": "^4.17.21"
+  },
```

Comment:
Not using this yet, but it's handy when you need it. 

It doesn't matter so much for Node apps, but protip for [lodash](https://lodash.com/docs/4.17.15), is to just import the module you want. 

So like instead of: 

```js
import _ from 'lodash'

if (_.isEqual(x, y)) {
  return
}
```
You'd do:

```js
import isEqual from 'lodash/isEqual'

if (isEqual(x, y)) {
  return
}
```

It's more than just a semantic difference of getting rid of the `_` prefix, the second method only imports the `isEqual` module instead of the entire `lodash` library. 

Again, not as important on the backend but very very important on the frontend and a good habit to get in to.